pybricksdev/cli/__init__.py

```diff
@@ -165,7 +165,9 @@ def add_parser(self, subparsers: argparse._SubParsersAction):
 
     async def run(self, args: argparse.Namespace):
         from ..ble import find_device
-        from ..connections import EV3Connection, PybricksHub, REPLHub
+        from ..connection.ev3 import EV3Connection
```

Comment:
I think I would call this module `ev3dev` instead of `ev3` since the connection only works with EV3's running ev3dev.

Reply:
Done.